src/App.js

```diff
@@ -1,26 +1,74 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react'
+import 'bootstrap/dist/css/bootstrap.css'
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox'
+import AddItem from './components/AddItem'
+import {Button} from 'react-bootstrap'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+
+  state = {
+    foodsList: foods,
+    showForm: false,
+    filteredList: foods,
+  }
+
+  handleAdd = (event) =>{
+    event.preventDefault()
+    console.log(event.target)
+    let newFood = {
+      id: this.state.foodsList.length +1,
```

Comment:
there is no use for this id property so no need to add it. If there was a delete functionality this might create ids tha are the same.